script/setup/prepare_env_windows.ps1

```diff
@@ -5,7 +5,7 @@
 # lived test environment.
 Set-MpPreference -DisableRealtimeMonitoring:$true
 
-$PACKAGES= @{ mingw = "10.2.0"; git = ""; golang = "1.18"; make = ""; nssm = "" }
+$PACKAGES= @{ mingw = "11.2.0"; git = ""; golang = "1.18.3"; make = ""; nssm = "" }
```

Comment:
oh! nice one; I definitely missed this one when grep'ing for the go version to update